### ISSUE TITLE
Supporting other platforms

### ISSUE THREAD

**Issue by Luphaestus** [Samsung-Loki/Thor#19]
Hi, 

First and foremost, great work! I was wondering if you were considering developing it for other platforms such as Windows.

**Comment by ethical-haquer** [comment]
> Hi,
> 
> First and foremost, great work! I was wondering if you were considering developing it for other platforms such as Windows.

From [here](https://xdaforums.com/t/dev-thor-flash-utility-the-new-samsung-flash-tool.4597355/page-3#post-89276959):

>  Can't be bothered to write a driver, for now.

And from [here](https://xdaforums.com/t/dev-thor-flash-utility-the-new-samsung-flash-tool.4597355/page-3#post-89278578):

> If anyone knows C#, you're free to make a fork.

**Comment by TheAirBlow** [comment]
Nope, this project is on the backburner as I don't have any devices to test on.
